include/exadg/incompressible_navier_stokes/spatial_discretization/operators/momentum_operator.cpp

```diff
@@ -102,8 +130,9 @@ MomentumOperator<dim, Number>::initialize(
   }
 
   // simply set pointers for convective and viscous kernels
-  this->convective_kernel = convective_kernel;
-  this->viscous_kernel    = viscous_kernel;
+  this->convective_kernel          = convective_kernel;
```

Comment:
with this initializer we get the `convective_kernel` and `viscous_kernel`. The variable viscosity models lie in `SpatialOperatorBase`